docs/docs-content/getting-started/additional-capabilities.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_label: "Palette Additional Capabilities"
+title: "Palette Additional Capabilities"
+description: "Learn more about Palette's Additional Capabilities."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["getting-started"]
+---
+
+Palette offers a range of additional capabilities designed to extend the functionality and versatility of Kubernetes
+workloads. These features comprise diverse deployment scenarios, ensuring organizations can manage their workloads
+across different environments. This can include data centers such as VMware vSphere and OpenStack, bare metal, or public
+cloud using your cloud accounts.
+
+Whether deploying workloads in remote locations with [Edge](./additional-capabilities.md#edge), securing sensitive
+information with [VerteX](./additional-capabilities.md#palette-vertex), maximizing management control with
+[self-hosted Palette](./additional-capabilities.md#self-hosted-palette), or integrating virtual machine workloads into
+Kubernetes environments with [Virtual Machine Orchestrator](./additional-capabilities.md#virtual-machine-orchestrator),
+Palette provides comprehensive solutions to meet different business needs. Learn more about each capability below.
+
+![A drawing of Palette with humans interacting](/getting-started/getting-started_additional-capabilities_palette.png)
+
+## Edge
+
+Palette Edge enables you to deploy Kubernetes workloads in remote or rural locations without having to provision
+high-bandwidth connectivity from these sites back to the cloud. Its secure and decentralized architecture allows
+services to operate even without connection to the central management plane while enforcing policies locally.
+
+Edge clusters are Kubernetes clusters set up on Edge hosts. These hosts can be bare metal or virtual machines located in
+isolated locations. Palette deploys and manages workload clusters at the Edge through the Palette management console,
+bringing computing and data storage closer to the source. This approach reduces latency and bandwidth issues that result
```

Comment:
```suggestion
isolated locations. Palette deploys and manages workload clusters at the Edge. You can manage Edge clusters locally on-site through the Edge Management console, or centrally through the Palette management plane. 
Palette Edge is able to meet your needs, regardless of the network topology your deployments face. 
```

We gotta be a bit careful here. Edge can be managed locally through the EMC (4.3) feature if in an airgap environment. If not, then you can use the default Palette UI.